build.cake

```diff
@@ -9,7 +9,7 @@ var configuration = Argument("configuration", "Debug");
 // SET PACKAGE VERSION
 //////////////////////////////////////////////////////////////////////
 
-var version = "2.1.0";
+var version = "2.1.1-b2";
```

Comment:
yes, i just used that for the version I uploaded to the reporter to check out.  I will remove this and make it 2.1.1 for the release